modules/cluster-logging-release-notes-5.1.0.adoc

```diff
@@ -1,8 +1,9 @@
 [id="cluster-logging-release-notes-5-1-0"]
 = OpenShift Logging 5.1.0
 
-This release includes:
+The following advisories are available for OpenShift Logging 5.2.x:
```

Comment:
Should that be `The following advisories are available for OpenShift Logging 5.1.x`?